flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/sink/filesystem/StreamingFileSink.java

```diff
@@ -291,6 +291,7 @@ T withBucketFactory(final BucketFactory<IN, BucketID> factory) {
 					bucketFactory,
 					new RowWisePartWriter.Factory<>(encoder),
 					rollingPolicy,
+					null,
```

Comment:
Add listener setter to this row wise part writer builder?

Reply:
Also added as previous one.